src/app/portlets/dot-edit-content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.spec.ts

```diff
@@ -138,6 +169,7 @@ describe('DotEditPageToolbarComponent', () => {
     });
 
     it('should disabled save button', () => {
+        component.page.locked = true;
         component.canSave = false;
```

Comment:
why do canSave can turn into false?, I think we should tigger the event (o events) necessary for turn it into false and not just set it